doc/configuring.rst

```diff
@@ -505,5 +505,5 @@ In general, conditional steps behave as any other regular steps, but here are so
     * Only one branch step will be executed
     * Both branches' artifacts will be checked for existence before the steps execution
     * Artifacts collection or any other side-effects will not be triggered for non-executed branch step
-    * If chosen branch step is not set, nothing will happen (e.g. conditional step failed, but ``Step.if_failed`` was not set)
+    * If chosen branch step is not set, nothing will happen. E.g. conditional step failed, but ``Step.if_failed`` was not set
```

Comment:
Nope. If you separate the sentence, you have to make it independent from the last one.
If I were writing these docs, I would put it like this:
```
For example, if the the ``Step.if_failed`` is not set and conditional step fails during the execution, nothing will happen.
```